osh/hub/other/test_fixtures/README.md

```diff
@@ -0,0 +1,23 @@
+## Fixtures howto
+
+The fixtures in this directory needs to be loaded manually in case needed.
+
+- users.json
```

Comment:
We can use `` for file names (and also for consistency with other `users.json` instances later):
```suggestion
- `users.json`
```